### PR TITLE
Feature add update article

### PR DESCRIPTION
**Pull request by Richarddesnoues**

### CODE REVIEW

File: public/assets/images/articles/65c21362ae9339.70709865.png
Comment (LuckyShuii) [inline public/assets/images/articles/65c21362ae9339.70709865.png:1]: ???

File: public/assets/js/admin/updateForm.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
-function previewImage(event) {
-    var reader = new FileReader();
-    reader.onload = function(){
-        var output = document.getElementById('preview');
-        output.src = reader.result;
-    };
-    reader.readAsDataURL(event.target.files[0]);
+const imageUpload = document.getElementById('imageUpload');
+console.log(imageUpload+'imageUpload');
```
Comment (LuckyShuii) [inline public/assets/js/admin/updateForm.js:2]: debug code to remove

File: public/assets/js/admin/updateForm.js [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+function previewImage(e) {
+   const input = e.target;
+   const image = document.getElementById('preview');
+   
+   console.log(input.files);
```
Comment (LuckyShuii) [inline public/assets/js/admin/updateForm.js:7]: debug code to remove

**Review by LuckyShuii — CHANGES_REQUESTED** [review]

File: src/View/Admin/components/_form_articleUpdate.html.twig [hunk 3]
```diff
@@ -103,4 +155,5 @@
 {% block javascript %}
 <script src="/assets/js/admin/updateForm.js"></script>
 <script src="/assets/js/admin/adminFormArticle.js"></script>
+>>>>>>> 6641439077193b08b1e5f3e061f6dafbcc003990
```
Comment (Benja-san) [inline src/View/Admin/components/_form_articleUpdate.html.twig:158]: git conflict leftovers here

File: src/View/Admin/components/_form_articleUpdate.html.twig [hunk 4]
```diff
@@ -35,25 +50,62 @@
...
+				<label for="actualImg">Image actuelle</label>
+				<img class="actualImg" src="/{{articleUpdate.imgSrc}}" alt="L'image n'existe pas dans le fichier !">
+			</div>
+		</div>
+		<div class="buttons">
+			<input id="validate" type="submit" value="Valider">
+			<a id="cancel" href="/admin/articles/">Annuler</a>
+		</div>
+	</form>
+=======
```
Comment (Benja-san) [inline src/View/Admin/components/_form_articleUpdate.html.twig:108]: git conflict leftovers here

**Review by Benja-san — CHANGES_REQUESTED** [review]
Did you think about option picker for article current author, status and category ?

**Review by Benja-san — APPROVED** [review]

**Review by LuckyShuii — APPROVED** [review]

**Pull request merged**